lib/breadcrumbs_on_rails/json_ld/builder.rb

```diff
@@ -20,7 +20,7 @@ def render_element(element, index)
           "@type" => "ListItem",
           "position" => index,
           "item" => {
-            "@id" => URI.join(@context.root_url, compute_path(element)).to_s,
+            "@id" => URI.join(@context.root_url, URI.escape(compute_path(element))).to_s,
```

Comment:
Thanks a million! Is it better to also wrap `@context.root_url` with `URL.escape`?
I hear we can use such non-ASCII characters in domain names.

When I set `config.action_controller.default_url_options = { host: 'https://Ü.com' }`, I got the following error. (which is the same as you got, right?)

```
URI::InvalidURIError:
  URI must be ascii only "http://\u00DC.com/"
```

Reply:
Yes, you are right. It is the same error. As I have no umlaut-domain, I did not checked this part. I think its like you said and it needs also the escape command.

Reply:
Okay, thanks, I'm merging your PR and will double-check as soon as possible if I should escape root_url also!